docs/book/src/topics/vpc/uploading-an-image.md

```diff
@@ -44,7 +44,7 @@ Service VPC Infrastructure Services in Resource_group <your_resource_group_or_ac
 target: service Cloud object storage in resource_group <your_resource_group_or_account>
 ```
 
-3) Go to https://cloud.ibm.com/vpc-ext/provision/customImage
+3) Go to [https://cloud.ibm.com/vpc-ext/provision/customImage](https://cloud.ibm.com/vpc-ext/provision/customImage)
```

Comment:
Previously:
<img width="553" alt="image" src="https://github.com/user-attachments/assets/c13f0e49-3f25-401e-9cb3-6314fb1e2a7f" />

Through the PR:
<img width="514" alt="image" src="https://github.com/user-attachments/assets/54361f87-77b8-4780-8828-37089837e4b4" />